### ISSUE TITLE
gradient color value will be optimized in the filter

### ISSUE THREAD

**Issue by chengwubin** [jakubpawlowicz/clean-css#945]
### Precheck
 
* Do a quick search and make sure a bug has not yet been reported;
* do a quick check if the bug still exists in the latest patch version;
* finally, be nice and have fun!

### Environment

* clean-css version - `npm ls clean-css`:
* node.js version - `node -v`:
* operating system:

### Configuration options

```js
var CleanCSS = require('clean-css');
new CleanCSS({
   compatibility: 'ie7'
})
```

### Input CSS

```css
/* the shorter the better! */
filter:progid:DXImageTransform.Microsoft.gradient(gradientType=0, startColorstr=#66000000, endColorstr=#66000000)}
```

### Actual output CSS

```css
filter:progid:DXImageTransform.Microsoft.gradient(gradientType=0, startColorstr=#60000, endColorstr=#60000)}
```

### Expected output CSS

```css
filter:progid:DXImageTransform.Microsoft.gradient(gradientType=0, startColorstr=#66000000, endColorstr=#66000000)}
```

**Comment by chengwubin** [comment]
but we can resolve it use this options:
new CleanCSS({
   compatibility: 'ie7,-properties.colors'
})

**Comment by jakubpawlowicz** [comment]
@chengwubin this is odd indeed - what's the 8-value hex? Is it #RRGGBBAA (a for alpha)?

**Comment by jakubpawlowicz** [comment]
Fixed in 4.1.5.
